src/extensions/mountainview/MVSortingUnitView/MVSortingUnitView.tsx

```diff
@@ -0,0 +1,44 @@
+import { Grid } from "@material-ui/core"
+import React, { Fragment, FunctionComponent } from 'react'
+import { Expandable } from "../../../containers/SortingView"
+import { SortingUnitViewProps } from "../../extensionInterface"
+import sortByPriority from "../../sortByPriority"
+
+
+const MVSortingUnitView: FunctionComponent<SortingUnitViewProps> = (props) => {
+    const plugins = {...props.plugins, sortingUnitViews: {...props.plugins.sortingUnitViews}}
+    // important to exclude this plugin (not only for this widget but for all children) to avoid infinite recursion
+    delete plugins.sortingUnitViews['MVSortingUnitView'] 
+    const sortingUnitViewPlugins = sortByPriority(plugins.sortingUnitViews)
+
+    return (
+        <Fragment>
+            {/* Non-full width first */}
```

Comment:
We split the plugin views into two categories: non-full width (p.fullWidth !== true) and full width (p.fullWidth === true). We display the non-full width first in hard-coded boxes of size 400x400. Then the full width views are displayed next, and they need to be expanded. Example of full width widget is the spike amplitude time view.